test/test.js

```diff
@@ -323,6 +323,63 @@ describe('csurf', function () {
     })
   })
 
+  describe.only('req.verifyToken()', function () {
+    var app, token, cookie;
+    beforeEach(function(done) {
```

Comment:
Do you have to re-make the app for every request, or can you just make it once with a "before"?